app/Http/Controllers/MapController.php

```diff
@@ -0,0 +1,126 @@
+<?php
```

Comment:
move this file into its own namespace and folder

Reply:
@Jellyfrog 
my intention was to create a base for all Maps, so own directory is not necessary and doubled code can be prevented.
Especially for rewriting NetworkMap Code, it only has to include another method into MapController Class

what do you think about this?

Reply:
I mean create a folder in Controllers named Maps and there u put all mapcontrollers. Check widget folder for inspiration

Reply:
changed :-)